### DISCUSSION TITLE
Add custom reading lists like "To read later"

### DISCUSSION THREAD

**Discussion by samlemarnais** [dailydotdev/daily#682] [Feature Requests]
I would like to have more lists than just the "bookmark" list.
That could be lists or tags (but the difference mostly depends on the UI actually).

For example, I'd like to store articles in "to read later", "for work", "for personal project X", "for client Y", "for stack JS" / PHP / Java, etc.

**Comment by idoshamun** [comment]
Hi @samlemarnais,

Thanks for the suggestion. Let's keep the discussion here: #520
